run_dir/static/js/data_del_plot.js

```diff
@@ -0,0 +1,360 @@
+window.current_plot_data = null;
+window.current_plot_obj = null;
+window.current_view_schemes = null;
+
+$(function(){
+    init_page_js();
+    refresh_plot();
```

Comment:
Why call refresh_plot() a second time, if we are already calling it in init_page_js() at line 348?

Reply:
Nice catch!